webapp/src/main/webapp/test/output.test.js

```diff
@@ -82,6 +82,54 @@ describe('Output', function() {
     expect(document.getElementById('error').innerHTML).toContain(output);
   });
   
+  /** Test for fast_travel_between_stops */
+ it('should issue fast travel between stops warning', function() {
+    const params = {
+      code: "fast_travel_between_stops",
+      totalNotices: 2,
+      notices: [
+        {
+          tripId: "trip1",
+          speedkmh: 4000,
+          stopSequenceList: [6,7]
+        },
+        {
+          tripId: "trip89",
+          speedkmh: 5000,
+          stopSequenceList: [1,2]
+        }
+      ]
+    };
+    fast_travel_between_stops(params);
+    const output = "<div><p class=\"warning\">Warning - Fast Travel Between Stops found!</p>\
+<p>Description: Travel speed between stops is very fast!.</p>\
+<p><b>2</b> found:</p>\
+<table>\
+<thead>\
+<tr>\
+<th>Trip ID</th>\
+<th>Travel Speed km/h</th>\
+<th>Stop Sequence</th>\
+</tr>\
+</thead>\
+<tbody>\
+<tr>\
+<td>trip1</td>\
+<td>4000</td>\
+<td>6,7</td>\
+</tr>\
```

Comment:
Nit: To help improve readability, can you possibly have the multiple columns of a single row in a single line?

Reply:
Sure

Reply:
Done now